### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)

### CODE REVIEW

**Review by catherinepope — APPROVED** [review]
👍

File: jekyll/_cci2_ja/first-steps.md [hunk 1]
```diff
@@ -35,12 +72,15 @@ CircleCI で最初のビルドを実行するには、[登録](https://circleci.
...
+    - CircleCI 上の一般的なオープンソースプロジェクトビルドを使って([React  by Facebook](https://app.circleci.com/pipelines/github/facebook/react))、または弊社独自のサンプルプロジェクト ([JavaScript サンプルアプリ](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/)や[Pyton サンプルアプリ](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/)) のいずれかを使用して、CircleCI アプリケーションをお試しください。 [パイプライン]({{ site.baseurl }}/2.0/pipelines/)や[ワークフロー]({{ site.baseurl }}/2.0/workflows) などの機能をご覧いただけます。 [Dashboard (ダッシュボード)] 、 [Projects (プロジェクト)] 、 [Organization Settings (組織の設定)] 、および [Plan (プラン)] のページは、 GitHub アカウントまたは Bitbucket アカウントを接続するまで使用できません。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/first-steps.md:75]:
```suggestion
    - CircleCI 上の一般的なオープンソースプロジェクトビルドを使って([React by Facebook](https://app.circleci.com/pipelines/github/facebook/react))、または弊社独自のサンプルプロジェクト ([JavaScript サンプルアプリ](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/)や[Python サンプルアプリ](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/)) のいずれかを使用して、CircleCI アプリケーションをお試しください。 [パイプライン]({{ site.baseurl }}/2.0/pipelines/)や[ワークフロー]({{ site.baseurl }}/2.0/workflows) などの機能をご覧いただけます。 [Dashboard (ダッシュボード)] 、 [Projects (プロジェクト)] 、 [Organization Settings (組織の設定)] 、および [Plan (プラン)] のページは、 GitHub アカウントまたは Bitbucket アカウントを接続するまで使用できません。
```

File: jekyll/_cci2_ja/plan-free.adoc [hunk 2]
```diff
@@ -34,4 +34,4 @@ Docker、 Linux、 Windows、および macOS のさまざまなリソースク
 
 === サポート
 
-https://discuss.circleci.com/[community forums]、 https://support.circleci.com/hc/ja[support portal]、および https://support.circleci.com/hc/ja/requests/new[global ticket-based system] からサポートをご利用いただけます。
\ No newline at end of file
+Scale プランでは、 https://discuss.circleci.com/[コミュニティフォーラム]、 https://support.circleci.com/hc/ja[サポートポータル]、および https://support.circleci.com/hc/ja/requests/new[グローバルチケットシステム] もご利用いただけます。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/plan-free.adoc:37]:
```suggestion
Scale プランでは、 https://discuss.circleci.com/[コミュニティフォーラム]、 https://support.circleci.com/hc/ja[サポートポータル]、および https://support.circleci.com/hc/ja/requests/new[チケットベースのグローバル サポート] もご利用いただけます。
```

haha, i am not too sure which expression is best but the suggestion about is based on what we label for the same thing on our 🇯🇵  Pricing page:

For your consideration!
Please feel free to reject or update accordingly :)

Screen Shot 2022-03-28 at 15 19 04

File: jekyll/_cci2_ja/plan-overview.adoc [hunk 3]
```diff
@@ -15,7 +15,7 @@ version:
 
 このドキュメントでは、開発者の皆様に向けた CircleCI の料金プランについてご案内します。
 
-NOTE: 現在、従来の CircleCI コンテナベースプランを利用されているお客様は、<<containers#,containers>> の使用についてのドキュメントを参照してください。 コンテナベースからクレジットベースへの切り替えをご希望のお客様は、[open a support ticket] (https://support.circleci.com/hc/ja/requests/new) よりサポートチケットを作成してお問い合わせください。
+NOTE: 現在、従来の CircleCI コンテナベースプランを利用されているお客様は、<<containers#,コンテナ>> の使用についてのドキュメントを参照してください。 コンテナベースからクレジットベースへの切り替えをご希望のお客様は https://support.circleci.com/hc/ja/requests/new[グローバルチケットシステム] よりサポートチケットを作成してお問い合わせください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/plan-overview.adoc:18]: same comment as previous re: `グローバルチケットシステム` :)

File: jekyll/_cci2_ja/troubleshooting-common-issues.md [hunk 4]
```diff
@@ -0,0 +1,1635 @@
...
+
+* **言語バージョンが異なる**： お使いのマシンと CircleCI で使用されている言語バージョンが同じであることを確認します。
+* **パッケージのバージョンが異なる**： 構成ファイルでパッケージのバージョンを明示的に指定します。
+* **タイムゾーンの問題**： 一部のテスト用フレームワークでは、モジュールがタイムゾーンに対応していない可能性があります。 パソコンで正しいタイムゾーンが設定されていないと、一部のテストが失敗する可能性があります。 詳細については、FAQ にある[この質問](https://circleci.com/docs/2.0/faq/#how-can-i-set-the-timezone-in-docker-images)を参照してください。
+* **ファイルの順序**： 一部のファイルシステムでは、ディレクトリごとに順序付けされたファイル構造が維持されます。 このため、すべてのファイルは実行のたびに同じ順序で読み取られます。 しかし、CircleCI のビルドコンテナのファイルシステムには順序がありません。 実施するテストが、一定の順序で実行した場合にのみ通るようなものであれば、CircleCI では失敗する可能性があります。
+* **「Out of Memory」エラー**： ビルドコンテナのプロセスによるメモリ使用量が多すぎる場合、 Linux の OOM killer により終了されることがあります。 If you are interested in recording your memory usage, visit [How to record a job's memory usage](https://support.circleci.com//hc/en-us/articles/360043994872).
+* **起動時間の違い**： Elasticsearch や PostgreSQL などのアプリケーションは、CircleCI で起動するときに多少の時間を要することがあります。 テストを実行する前に、サービスが利用可能になるまで明示的に待機することで、この問題を修正できます。
+* **信頼性の低いサービス**： テストでサードパーティー製のサービスを使用する場合、そのサービスが実際に実行されていることを確認してください。
+
+もし問題が解決できない場合、[サポートチケット](https://support.circleci.com/hc/en-us)を登録してください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/troubleshooting-common-issues.md:126]:
```suggestion
もし問題が解決できない場合、[サポートチケット](https://support.circleci.com/hc/ja)を登録してください。
```

File: jekyll/_cci2_ja/troubleshooting-common-issues.md [hunk 5]
```diff
@@ -0,0 +1,1635 @@
...
+OCI runtime create failed: container_linux.go:345:   
+starting container process caused "process_linux.go:303:   
+getting the final child's pid from pipe caused \"EOF\"": unknown
+```
+
+This usually occurs due to a bad [command](https://circleci.com/docs/2.0/configuration-reference/#docker) being passed to one of the secondary service containers in your job.
+
+このエラー メッセージの原因は、セカンダリ サービス コンテナがクラッシュしたか、予期せず終了したことです。 これは、CircleCI が Docker デーモンから PID を取得できず、エラーが発生して終了したことを意味します。
+
+コンテナを正常にスピンアップするには、コマンドとエントリポイントが正しいことを確認してください。 [CircleCI CLI](https://circleci.com/docs/ja/2.0/local-cli/#%E3%83%9E%E3%82%B7%E3%83%B3%E4%B8%8A%E3%81%AE%E3%82%B3%E3%83%B3%E3%83%86%E3%83%8A%E5%86%85%E3%81%A7%E3%81%AE%E3%82%B8%E3%83%A7%E3%83%96%E3%81%AE%E5%AE%9F%E8%A1%8C) を使用すると、ローカルでのテストにより、ジョブをローカルで実行してすべての Docker コンテナが正常にスピンアップするかどうかを確認できます。 エラーを解決するには、お使いの設定ファイルで定義している Docker イメージのドキュメントを参照して、スピンアップ時に使用できるコマンドを確認してください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/troubleshooting-common-issues.md:252]:
```suggestion
コンテナを正常にスピンアップするには、コマンドとエントリポイントが正しいことを確認してください。 [CircleCI CLI](https://circleci.com/docs/ja/2.0/local-cli/#run-a-job-in-a-container-on-your-machine) を使用すると、ローカルでのテストにより、ジョブをローカルで実行してすべての Docker コンテナが正常にスピンアップするかどうかを確認できます。 エラーを解決するには、お使いの設定ファイルで定義している Docker イメージのドキュメントを参照して、スピンアップ時に使用できるコマンドを確認してください。
```

Found the right link @ https://circleci.com/docs/ja/2.0/local-cli/#run-a-job-in-a-container-on-your-machine 🤖

**Review by kelvintaywl — COMMENTED** [review]
Almost LGTM, except some tiny fixes / suggestions!

thanks for the hard work! 🍶

File: jekyll/_cci2_ja/plan-free.adoc (hunk 2 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/plan-free.adoc]: Thank you for pointing this out! I'll update other pages in next PR.

**Review by nanophate — APPROVED** [review]
Thank you for updating the request

**Pull request merged**